File: client/src/main/java/xyz/beriholic/beeyes/client/entity/RuntimeInfo.java
package xyz.beriholic.beeyes.client.entity;

import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class RuntimeInfo {
    long timestamp;
    double cpuUsage;
    double memoryUsage;
    double diskUsage;
    double networkUploadSpeed;
    double networkDownloadSpeed;
    double diskWriteSpeed;
    double diskReadSpeed;
}

File: client-go/internel/model/runtime_info.go
package model

type RuntimeInfo struct {
	Timestamp            int64   `json:"timestamp"`
	CpuUsage             float64 `json:"cpuUsage"`
	MemoryUsage          float64 `json:"memoryUsage"`
	DiskUsage            float64 `json:"diskUsage"`
	NetworkUploadSpeed   float64 `json:"networkUploadSpeed"`
	NetworkDownloadSpeed float64 `json:"networkDownloadSpeed"`
	DiskWriteSpeed       float64 `json:"diskWriteSpeed"`
	DiskReadSpeed        float64 `json:"diskReadSpeed"`
}
